ludwig/models/llm.py

```diff
@@ -95,13 +100,16 @@ def __init__(
         _, self.output_feature_decoder = self.output_features.items()[0]
 
         # ================ Combined loss metric ================
+        ####SXK should we have a class to capture all the loss metrics and output to a dashboard during a run
```

Comment:
We already do that using a set of predefined metric function classes. These are just maintained separately